nodl_python/nodl/types/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the
+# GNU Limited General Public License version 3, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+from ._types import (  # noqa: F401
+    Action,
+    Node,
+    NoDLData,
+    NoDLInterface,
+    Parameter,
+    Service,
+    Topic,
+)
```

Comment:
What exactly are we gaining by doing this as opposed to leaving `types.py` as a module? I only see this making sense if we want to split it into multiple files.

Reply:
Reverted in 97698c7